lib/statsample-glm/glm/formula.rb

```diff
@@ -29,9 +29,11 @@ def parse_formula(form = :token)
 
       private
 
-      def add_contant_term_if_required
-        # TODO: Add support for constants
-        @tokens.unshift Token.new('1')
+      def manage_contant_term
+        @tokens.unshift Token.new('1') unless
+          @tokens.include?(Token.new('1')) ||
+          @tokens.include?(Token.new('0'))
+        @tokens.delete Token.new('0')
```

Comment:
Thanks @zverok 

I implemented it in this manner:

```ruby
      TOKEN_1 = Statsample::GLM::Token.new('1')
      def self.Token val
        return TOKEN_1 if val == '1'
        Token.new(val)
      end
```
But it gives, this error:
```ruby
/home/ubuntu/workspace/statsample-glm/lib/statsample-glm/glm/formula.rb:59:in `<class:FormulaWrapper>': uninitialized constant Statsample::GLM::Token (NameError)
```